robot/basestation/stream_capture.py

```diff
@@ -8,15 +8,16 @@
 from robot.util.utils import run_shell
 import robot.basestation.ros_utils as ros_utils
 from robot.basestation.ros_utils import fetch_ros_master_ip
+import ffmpeg
 
 global proc_video
```

Comment:
I know you didn't create this variable but could you rename it to be more clear? Something like `video_process`

Reply:
Another minor issue is that this here is not the correct usage of `global` keyword. AFAIK in most other languages that I have experienced this is the normal approach, but in Python the way it should work is that we declare variables normally on the outter most scope (without the `global` keyword) and then in inner scopes (so like in function definitions) we assert `global` before the variable so that it's known that we are referring to the global variable. We use the keyword in local scopes for both read/writing to the variable. Here's a [reference](https://www.programiz.com/python-programming/global-keyword).

Basically, you'd have to move this line (13) into the top of the functions that make use of this variable.

Reply:
Yeah, I remember reading that in Slack. I am changing that